lib/selenium-cucumber/click_elements_steps.rb

```diff
@@ -1,9 +1,12 @@
 require_relative 'methods/click_elements_methods'
 
-# click on web element
-When(/^I click on element having (.+) "(.*?)"$/) do |type, access_name|
+When(/^I click on element having (id|name|class|xpath|css) "(.*?)"(?: and text "(.*?)")?$/) do |type, access_name, text|
```

Comment:
you can use (.+) instead (id|name|class|xpath|css) since we are validating locator type.